docs/preflight/README.md

```diff
@@ -47,38 +47,46 @@ This `UID` is particularly useful to perform cleanup of resources created during
 
 The following checks are included in preflight:
 
-1. `check-kubectl` - Ensures **kubectl** utility is present on system
-2. `check-kubectl-access` - Ensures **kubectl** is pointed to k8s cluster (i.e can access the remote target cluster)
+1. `check-kubectl` - Ensures **kubectl** utility is present on system. This check is skipped if `--in-cluster` flag is enabled.
+
+2. `check-kubectl-access` - Ensures **kubectl** is pointed to k8s cluster (i.e can access the remote target cluster). This check is skipped if `--in-cluster` flag is enabled.
```

Comment:
This check is not skipped for in-cluster execution

Reply:
so `check-kubectl-access` is a little misleading since when it runs inside a container, it's not using kubectl utility. changed it to `check-cluster-access` which makes more sense.